src/components/mortgage-investment-compare/index.js

```diff
@@ -88,16 +88,20 @@ class MortgageInvestmentCompare extends React.Component {
   // Returns future value dynamically depending on mortgage term length.
   calculateFV (option1, option2, state) {
     // TODO: what if terms of option1 and option2 are equal?
+    const inflationAdjustedReturn =
+      (1 + state.investmentRate / COMPOUND_FREQUENCY / 100) /
+        (1 + state.inflation / COMPOUND_FREQUENCY / 100) -
+      1
```

Comment:
Oh hey! It looks like you also figured it out for the batch calculation! 🎉

Reply:
yeah, but it's pretty much the same as that we had before. I'll explain when we meet about this